### DISCUSSION TITLE
How can I delete the entire database?

### DISCUSSION THREAD

**Discussion by mxvsh** [tekartik/sembast.dart#372] [Q&A]
I want to delete everything that's in the store after user logs out from the application.

I want to delete all the data not just one store, because I am generating store names from IDs from the backend.

**Comment by alextekartik** [comment]
You can use [`deleteDatabase()`](https://pub.dev/documentation/sembast/latest/sembast/DatabaseFactory/deleteDatabase.html) to delete an entire database. You can use [`delete()`](https://pub.dev/documentation/sembast/latest/sembast/SembastStoreRefExtension/delete.html) to delete a single store.
